observability/monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -191,16 +191,18 @@ endif::openshift-dedicated,openshift-rosa[]
 include::modules/monitoring-configuring-remote-write-storage.adoc[leveloffset=+1]
 include::modules/monitoring-supported-remote-write-authentication-settings.adoc[leveloffset=+2]
 include::modules/monitoring-example-remote-write-authentication-settings.adoc[leveloffset=+2]
+include::modules/monitoring-example-remote-write-queue-configuration.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 .Additional resources
-
-* See link:https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage[Setting up remote write compatible endpoints] for steps to create a remote write compatible endpoint (such as Thanos).
-* See link:https://prometheus.io/docs/practices/remote_write/#remote-write-tuning[Tuning remote write settings] for information about how to optimize remote write settings for different use cases.
+ifndef::openshift-dedicated,openshift-rosa[]
+* xref:../../rest_api/monitoring_apis/prometheus-monitoring-coreos-com-v1.adoc#spec-remotewrite-2[Prometheus REST API reference for remote write]
+endif::openshift-dedicated,openshift-rosa[]
+* link:https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage[Setting up remote write compatible endpoints]
+* link:https://prometheus.io/docs/practices/remote_write/#remote-write-tuning[Tuning remote write settings]
```

Comment:
[When pointing to a link that exists external to Red Hat’s customer portal documentation](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute), add the link attribute, and use the <document_heading_name> (<document_source>) format.

```suggestion
* link:https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage[Setting up remote write compatible endpoints] (Prometheus documentation)
* link:https://prometheus.io/docs/practices/remote_write/#remote-write-tuning[Tuning remote write settings] (Prometheus documentation)
```

Reply:
Oh I never knew this rule. Thank you for letting me know!